server/mentionscrawler/authentication/authenticate.py

```diff
@@ -29,6 +29,6 @@ def wrapper(*args, **kwargs):
 
                 return fn(user=user, *args, **kwargs)
 
-            return error_response("No user logged in. Please log in again.")
+            return error_response("No user logged in. Please log in again.", "Unauthorized")
```

Comment:
I like this better than what I changed it to in my code XD

Reply:
I only added this because error_response took two arguments

Reply:
ya, I had forgotten I changed it to that, and hadn't updated it on my end either XD